src/Type/Accessory/AccessoryNonEmptyStringType.php

```diff
@@ -144,6 +145,11 @@ public function toFloat(): Type
 		return new FloatType();
 	}
 
+	public function toBoolean(): BooleanType
```

Comment:
You can use UndecidedBooleanTypeTrait instead and remove the truthy one of course